### ISSUE TITLE
Reproduce the results for LOMO 

### ISSUE THREAD

**Issue by shawnricecake** [OpenLMLab/LOMO#66]
Hi,

I tried the LOMO finetune on WIC dataset with the giving config [red line], 
I also tried to change the batch size from 16 (*8GPU) to 32 (*8GPU) [green line], and further the seed from 42 to 6 [purple line],
and the training loss and eval loss are the following:

<img width="1226" alt="image" src="https://github.com/OpenLMLab/LOMO/assets/39080134/96b19979-8200-463e-bf54-eae3087c8664">

<img width="1243" alt="image" src="https://github.com/OpenLMLab/LOMO/assets/39080134/a4be3f5a-a857-493a-8797-813c5ba46a6d">

**Comment by KaiLv69** [comment]
We show the hyperparameters in our paper. FYI:
<img width="572" alt="image" src="https://github.com/OpenLMLab/LOMO/assets/39761308/a2f30013-e02c-4b59-b486-192e736979da">

**Comment by shawnricecake** [comment]
> We show the hyperparameters in our paper. FYI: <img alt="image" width="572" src="https://user-images.githubusercontent.com/39761308/279570370-a2f30013-e02c-4b59-b486-192e736979da.png">
> 
> The batch size refers to the total batch size instead of batch size per gpu.

Hi,

Thanks, I find that the batch size should be the global batch size and I got the results at 0.70533

Thanks for your patience
Xuan
